pkg/services/sqlstore/alert_notification.go

```diff
@@ -185,7 +185,7 @@ func getAlertNotificationUidInternal(query *models.GetAlertNotificationUidQuery,
 	}
 
 	if len(results) == 0 {
-		return fmt.Errorf("Alert notification [ Id: %v, OrgId: %v ] not found", query.Id, query.OrgId)
+		return models.ErrAlertNotificationFailedTranslateUniqueID
```

Comment:
Looking at the code I'm unsure if any other consumer than `NewRuleFromDBAlert` are calling this. Since the name of the function is `getAlertNotificationUidInternal` maybe the returned error could reuse the existing `models.ErrAlertNotificationNotFound`. If there's only one consumer of this function I guess we can stick with returning `models.ErrAlertNotificationFailedTranslateUniqueID`. WDYT?

Reply:
@marefr 
do you still want to return `models.ErrAlertNotificationNotFound` here? personally, I would prefer to differentiate them.

Reply:
Let's stick with `ErrAlertNotificationFailedTranslateUniqueID` for now